packages/svelte/src/internal/client/dom/template.js

```diff
@@ -94,12 +94,13 @@ export function template_with_script(content, flags) {
 /**
  * @param {string} content
  * @param {number} flags
+ * @param {'svg' | 'math'} ns
  * @returns {() => Node | Node[]}
  */
 /*#__NO_SIDE_EFFECTS__*/
-export function svg_template(content, flags) {
+export function ns_template(content, flags, ns = 'svg') {
 	var is_fragment = (flags & TEMPLATE_FRAGMENT) !== 0;
-	var fn = template(`<svg>${content}</svg>`, 0); // we don't need to worry about using importNode for SVGs
+	var fn = template(`<${ns}>${content}</${ns}>`, 0); // we don't need to worry about using importNode for SVGs
```

Comment:
I assume this is also the case for `math` namespace stuff? Could the comment be updated?

Reply:
```suggestion
	var fn = template(`<${ns}>${content}</${ns}>`, 0); // we don't need to worry about using importNode for namespaced elements
```